### DISCUSSION TITLE
UI Live View points to small mjpeg stream without parameters

### DISCUSSION THREAD

**Discussion by parkroyk** [roflcoopter/viseron#588] [Q&A] [answered]
It would be great to be able to click Live View in the UI and be able to view the full-resolution video of the camera. I tried configuring the static stream, but that didn't do anything.

      mjpeg_streams:
        camera_1_stream:
          width: 3072
          height: 2048

Am I missing something from my config?

**Comment by roflcoopter** [comment, marked as answer]
You are correct, the live view right now are the frames processed by Viseron, so if you are using a low res substream the quality will be pretty bad.

I am working on 24/7 recordings right now, but after that I will improve the live view

**Reply by madman2012** [reply]
Will this be improved in v3? Why couldn't it just pass the RTSP stream to the browser?

**Reply by roflcoopter** [reply]
My plan is to fix this in 3.1, need to release 3.0 soon since it has been in development for so long.
From my knowledge browsers generally cant play RTSP streams directly so it has to be proxied using something like WebRTC
